src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -298,21 +298,14 @@ private ParseUserDefinitionResult ParseUDFsAndNamedFormulas(string script, Parse
                     _curs.TokMove();
                     ParseTrivia();
 
-                    if (_curs.TidCur == TokKind.Semicolon)
+                    if (_curs.TidCur == TokKind.Semicolon || _curs.TidCur == TokKind.Eof)

```

Comment:
> || _curs.TidCur == TokKind.Eof [](http://example.com/codeflow?start=57&length=31)

This changes the public grammar for named formulas (no longer requiring semicolon at end)

consider separate PR? Ensure approvals.